dags/openshift_nightlies/scripts/install/jetpack.sh

```diff
@@ -66,7 +66,7 @@ run_jetpack(){
         # disable openshift install as we already installed via jetpack
         export OPENSHIFT_INSTALL=false
         export DYNAMIC_DEPLOY_PATH=$DEPLOY_PATH
-        time /home/airflow/.local/bin/ansible-playbook -i ../inventory -vv OCP-4.X/install-on-osp.yml --extra-vars "@${json_file}" | tee $(date +"%Y%m%d-%H%M%S")-post-install.timing
+        time /home/airflow/.local/bin/ansible-playbook -i ../inventory -vv OCP-4.X/install-on-osp.yml -e openshift_toggle_workload_node=false --extra-vars "@${json_file}" | tee $(date +"%Y%m%d-%H%M%S")-post-install.timing
```

Comment:
@masco why pass this var individually? We can pass it with other --extra-vars in install.json

Reply:
we need this var as `true` on jetpack, since we are creating the workload nodes from jetpack.